website/src/components/Visualisations/Resources/LineGraph.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Brush, ResponsiveContainer, LineChart, Line, YAxis, XAxis, CartesianGrid, Tooltip, Legend, Surface, Symbols, ReferenceLine } from 'recharts';
+import outputJSON from '../../../output/output.json';
+import _ from "lodash";
+
+
+class LineRechartComponent extends React.Component<{}, any> {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            disabled: [],
+            lineColours: {
+                team1: "#0095FF",
+                team2: "#FF0000",
+                team3: "#802FF0",
+                team4: "#00C49F",
+                team5: "#FFBB28",
+                team6: "#FF8042",
+            },
+            datapaths: {
+                team1: "ClientInfos.Team1.Resources",
+                team2: "ClientInfos.Team2.Resources",
+                team3: "ClientInfos.Team3.Resources",
+                team4: "ClientInfos.Team4.Resources",
+                team5: "ClientInfos.Team5.Resources",
+                team6: "ClientInfos.Team6.Resources",
+            },
+            chartData: outputJSON
+        };
+    }
+
+    handleClick = dataKey => {
+        const { disabled } = this.state;
+        if (disabled.includes(dataKey)) {
+            this.setState({
+                disabled: this.state.disabled.filter(obj => obj !== dataKey)
+            });
+        }
+        else {
+            this.setState({
+                disabled: this.state.disabled.concat(dataKey)
+            });
+        }
```

Comment:
```suggestion
this.setState({
      disabled: this.state.disabled.includes(dataKey)
        ? this.state.disabled.filter((obj) => obj !== dataKey)
        : this.state.disabled.concat(dataKey),
    });
```

My indentation is broken but you get the point